packages/jsii-pacmak/test/expected.jsii-calc/java/src/main/java/software/amazon/jsii/tests/calculator/derived_class_has_no_properties/Base.java

```diff
@@ -1,4 +1,4 @@
-package software.amazon.jsii.tests.calculator.DerivedClassHasNoProperties;
+package software.amazon.jsii.tests.calculator.derived_class_has_no_properties;
```

Comment:
Why is this happening?

Reply:
Because `DerivedClassHasNoProperty` is a namespace... so it gets case-adjusted like it should.

Reply:
got it. ok